examples/co2_and_tracer_analysis.py

```diff
@@ -126,7 +129,7 @@ def determine_tracer(img: np.ndarray, base: np.ndarray) -> np.ndarray:
 
 # Plot
 plt.imshow(tracer)
-plt.show(block = False)
+plt.show(block=False)
```

Comment:
This file has been changed in main now. I do not have strong opinions, but it s important for pytest that the plots eventually closed themselves, unless we can get a pytest-flag of some sort